org-transclusion.el

```diff
@@ -796,6 +797,15 @@ It is meant to be used by
     (user-error "Error.  Link in #+transclude is mandatory at %d" (point))
     nil))
 
+(defun org-transclusion-keyword-value-thing-at-point (string)
```

Comment:
I am under the impression that `org-transclusion-keyword-value-thing-at-point` is intended for source code file (?). If so, this `defun` can be moved to `org-transclusion-src-lines.el` file and we can remove the `declare-function` at the top.

Reply:
I think you're right, I'll take a look.